content/network/_index.md

```diff
@@ -139,8 +139,9 @@ A container receives an IP address out of the IP subnet of the network.
 The Docker daemon performs dynamic subnetting and IP address allocation for containers.
 Each network also has a default subnet mask and gateway.
 
-When a container starts, it can only attach to a single network, using the `--network` flag.
-You can connect a running container to additional networks using the `docker network connect` command.
+You can connect a running container to multiple networks,
```

Comment:
I think it'd be valuable to detail how the advanced `--network` format can be used to attach to multiple networks. I think the doc doesn't mention that right now. Here's the relevant PRs:

- https://github.com/docker/cli/pull/4419
- https://github.com/docker/cli/pull/1767

I think it might worth also mentioning how the CLI interprets top-level flags when multiple networks are specified.